src/03-control-flow.js

```diff
@@ -179,7 +236,11 @@ function evenOrOdd(a) {}
  *  evenOrOddWithTernary(8);
  *  //> "Even"
  */
-function evenOrOddWithTernary(a) {}
+function evenOrOddWithTernary(a) {
+  let result =
+    (a % 2 === 0) ? "Even" : "Odd";
```

Comment:
Remember we actually do not need our `  ( ) ` here.  The ternary is only one operation.